pages/api/system/reload.js

```diff
@@ -234,28 +234,68 @@ export default async function handler(req, res) {
       }
 
       // - events
-      try {
-        if (profile.events) {
+      async function getCoordinates(city, state, country) {
+        let locationDb = {};
+        const provided = [city, state, country].filter((x) => x).join(",");
+        if (locationDb[provided]) {
+          return locationDb[provided];
+        }
+        try {
+          const location = await fetch(
```

Comment:
There is this functionality already, maybe we can move the existing functionality into a separate function for reused
https://github.com/EddieHubCommunity/LinkFree/blob/main/services/github/getLocationByUsername.js#L36

Reply:
So, you are saying that similar to `getLocationByUsername` we should create `getLocationByAddress` function and pass `city`, `state` and `country`?

Reply:
We can abstract the "location" part and make it more generic so it can be used in both places - I am travelling for the next 3 days with limited internet, but I can help when back if you need - thank you for pushing this forwards, it was a big issue

Reply:
I think I will need your help as I didn't understood what should I do for this. 
Thank you for your compliment.

Reply:
Sure 👍 let's do it 💪 I am still travelling, but I can take a better look in a few days

Reply:
Hey, have you taken a look? I have completed all other suggestions.

Reply:
I am on it at the moment 👍 I think I will be on it again tomorrow also, you did a lot of great work 💪 but I would like to make some small tweaks - I might have some questions / ideas to bounce with you, are you in the EddieHub Discord?

Reply:
Where can I find the link to join the discord ?

Reply:
In my bio, but here it is http://discord.eddiehub.org